st2client/st2client/commands/action.py

```diff
@@ -1462,6 +1486,13 @@ def __init__(self, resource, *args, **kwargs):
         self.parser.add_argument(
             "id", help=("ID of the %s." % resource.get_display_name().lower())
         )
+        self.parser.add_argument(
+            "-x",
```

Comment:
Can we get a long form argument as well?

```suggestion
            ["-x", "--exclude-result"],
```

Reply:
Yep